### DISCUSSION TITLE
Idées d’améliorations pour la partie tutos de Skyfleet.blue

### DISCUSSION THREAD

**Discussion by Leandraa13** [skyfleet-blue/skyfleet#1] [Ideas]
Voilà quelques modifications que je voudrais faire sur la partie tuto afin de facilité la visibilité et améliorer les pages. Dites moi ce que vous en pensez?  (J'ai fait un AVANT - A FAIRE - APRES pour vous montrer ce que ça pourrait donner comme modifications)

###  **AVANT** Tutoriels Bluesky
- Comment combiner un newsbot RSS pour générer des Customs Feeds thématiques ?
- Comment créer un custom feed à signets
- Comment créér un feed combiné ? 
- Comment faire un custom feed combine avec 2 listes in/out
- Comment faire un custom feed à partir d’un fauses mute-lit?
- Comment segmenter l'activité d'un compte sur bluesky par langue ?
- Comment supprimer un custom feed ?
- Comment utiliser le Jazbot ?
- Comment épingler un post dans un Feed ?
- Feed Generator
- Filtres V1

### **A FAIRE** To do list :

- [x] Renommer les pages en enlevant « Comment » afin de raccourcir les titres (améliorer la lisibilité)

- [x] Ajouter la question en titre sur la page pour que les utilisateurs sachent ce que va expliquer la page 

- [ ] Combiner certaines pages qui sont un peu redondantes 
Comment faire un custom feed combine avec 2 listes in/out  **+** Comment faire un custom feed à partir d’un fauses mute-lit? **+** Filtre V1 
= Utiliser les mute list pour faire un custom feed 

### **APRES** Tutoriels Bluesky
- Combiner un newsbot RSS pour générer des Customs Feeds thématiques 
- Créer un custom feed à signets
- Créér un feed combiné 
- Utiliser les mute liste pour faire un custom feed 
- Segmenter l'activité d'un compte sur bluesky par langue avec un custom feed
- Supprimer un custom feed 
- Utiliser le Jazbot 
- Epingler un post dans un Feed 
- Générateur de feed

### Questions : 
•	Les tutos sont rangés par ordre alphabétiques :  Est ce qu’il y aurait la possibilité de les ranger dans un ordre précis ? Sinon, est ce qu'on pourrait les numéroter pour que ça apparaisse dans un ordre un peu plus logique? Exemple : 
1- Créer un custom feed à signets
2- Segmenter l'activité d'un compte sur bluesky par langue avec un custom feed 
3- Utiliser les mute list pour faire un custom feed 
4- Créer un feed combiné ? 
5- Epingler un post dans un Feed 
6- Supprimer un custom feed 
7- Utiliser le Jazbot 
8- Combiner un newsbot RSS pour générer des Customs Feeds thématiques 
9- Générateur de feed

•	Est-ce que vous avez d’autres tutos qu’il faudrait mettre en place pour qu'on se fasse une liste des tutos à faire ?

**Comment by rmdes** [comment]
D'accord avec tout !!

Pour la question: oui il y a moyen, mais alors ça implique que pour chaque page qu'on crée on doit aller dans le menu et définir où elle s'affiche, ce n'est pas un soucis, c'est juste une manipulation en plus et ça implique d'utiliser un des modes de menu de navigation qui sert justement à faire ça 😄

**Reply by Leandraa13** [reply]
D'accord donc je peux d'abord faire les modification de titres et en ensuite on les rangera dans un ordre précis

**Comment by rmdes** [comment]
Pour le todo point 3, en fait "[Comment faire un custom feed à partir d’un fauses mute-list](https://skyfleet.blue/fr/tutoriels/fausse-mute-list)" provient du blog et "Comment faire un custom feed combine avec 2 listes in/out " est tout récent, il y aussi un "doublons" de tutoriels entre [ça](https://skyfleet.blue/fr/tutoriels/feed-combine-mute) et [celui ci](https://skyfleet.blue/fr/tutoriels/feed-combine) parce que le cas de figure n'est pas tout à fait le même, il faudrait sûrement le renommer aussi pour qu'il soit plus facile à distinguer de l'autre ?

**Reply by Leandraa13** [reply]
Je peux  faire une page générique sur "comment utiliser les mute list" et dans cette page lister les exemples  de Najat et de la chute pour avoir les tutos précis pour ces cas deux figures

**Reply by rmdes** [reply]
ça me paraît une bonne idée !!

**Comment by Mwyann** [comment]
Je suis d'accord avec tout, et quitte à parler d'arborescence, je me demande s'il ne faudrait pas créer un dossier spécifique pour les pages qui concernent les feeds (sous la rubrique tutos éventuellement) afin de séparer les tutos feeds (qui sont majoritaires) des tutos Bluesky "génériques" (flux RSS, jazbot...)

**Reply by rmdes** [reply]
Oui je vois ce que tu veux dire, il faudrait voir comment organiser ça, jusqu'ici "custom feeds" listait des custom feeds et l'annuaire, peut être il faut tour réorganiser en fonction du nouveau menu, ça devient encore plus clair avec celui ci activé 
![image](https://github.com/skyfleet-blue/skyfleet/assets/12155003/a0f095cf-a632-4457-ae08-7b4ce2768361)

**Reply by rmdes** [reply]
Pour l'instant on peut encore passer d'un menu à l'autre, mais une fois le menu organisé et bien dessiné on peut juste avoir le nouveau menu sans le bouton "parcourir" ci dessus, ça va être lisible pour le public si on garde que le menu dessiné sur mesure non?

**Reply by rmdes** [reply]
Je me pose aussi des questions sur la nécéssite de la page/section Astuces, ça avait du sens sur le blog, ça me permettait de faire des posts en donnant un hint vers quelque chose de précis à obtenir mais ici avec un wiki je suis pas convaincu, beaucoup de choses qui sont sur la page [parente](https://skyfleet.blue/fr/tips) **astuces** sont déjà dans **Outils**, par contre les sous pages de **Astuces** ne sont pas vraiment des tutoriels...il faudrait peut être revoir comment on appelle ça

**Reply by Leandraa13** [reply]
Je trouve le nouveau menu vachement mieux comme ça. Ca permets plus facilement d'acceder au grandes catégories. Après j'enleverais pas forcément le bouton "parcourir" car je trouve que c'est une autre façon de chercher l'info qui peut être intéressante (a voir dans le futur). 
Tu as raison il va aussi falloir revoir aussi les sections **Astuces/outils** pour essayer d'avoir moins d'info disséminé à droite et à gauche. Je vais y réflechir
